app/experimenter/nimbus-ui/src/components/FormOverview/index.tsx

```diff
@@ -120,233 +130,217 @@ const FormOverview = ({
+                Risk Mitigation Checklist Link
+                {isMissingField!("risk_mitigation_link") && (
+                  <InlineErrorIcon
+                    name="risk_mitigation_link"
+                    message="A Risk Mitigation Checklist is required"
+                  />
+                )}
+              </Form.Label>
+              <Form.Control
+                {...formControlAttrs("riskMitigationLink", {})}
+                type="url"
               />
-              <ReactTooltip />
-            </Form.Label>
-            <div>
-              {defaultValues.documentationLinks.map((docLink, docIdx) => (
-                <Form.Group
-                  className="mb-0"
-                  data-testid="DocumentationLink"
-                  key={`doc-link-${docIdx}`}
+              <Form.Text className="text-muted">
+                Go to the{" "}
+                <LinkExternal href={EXTERNAL_URLS.RISK_MITIGATION_TEMPLATE_DOC}>
+                  risk mitigation checklist
+                </LinkExternal>{" "}
+                to make a copy and add the link above
+              </Form.Text>
+              <FormErrors name="riskMitigationLink" />
+            </Form.Group>
+
+            <Form.Group controlId="documentationLinks">
+              <Form.Label>
+                Additional Links
+                <Info
+                  data-tip={DOCUMENTATION_LINKS_TOOLTIP}
+                  data-testid="tooltip-documentation-links"
+                  width="20"
+                  height="20"
+                  className="ml-1"
+                />
+                <ReactTooltip />
+              </Form.Label>
+              <div>
+                {documentationLinks.map((documentationLink, index) => (
+                  <FormDocumentationLink
+                    key={documentationLink.id}
+                    {...{
+                      documentationLink,
+                      setSubmitErrors,
+                      fieldNamePrefix: `documentationLinks[${index}]`,
+                      submitErrors:
+                        (submitErrors?.documentation_links &&
+                          (submitErrors?.documentation_links as SubmitErrorRecord[])[
+                            index
+                          ]) ||
+                        {},
+                      //@ts-ignore react-hook-form types seem broken for nested fields
+                      errors: (errors?.documentationLink?.[index] ||
+                        {}) as FormDocumentationLinkProps["errors"],
+                      //@ts-ignore react-hook-form types seem broken for nested fields
+                      touched: (touched?.documentationLink?.[index] ||
+                        {}) as FormDocumentationLinkProps["touched"],
+                      onRemove: () => {
+                        removeDocumentationLink(documentationLink);
+                      },
+                      canRemove: index !== 0,
```

Comment:
You `canRemove` this now right? (see what I did there)

Reply:
It could be removed but I think I'll leave it in as the [Figma designs](https://www.figma.com/file/5SLa07XqQb0ttB1ay1gami/Nimbus-Console-UI?node-id=218083%3A23) indicate it should be this way. We could discuss removing it though, for sure, as it could be potentially confusing UX.

Reply:
Ahh I see. I didn't realize that was per design. I suppose we could slap a `margin-right` on the first input to better match.